src/style.css

```diff
@@ -1,3 +1,71 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  --header-link-color: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+img {
+  display: block;
+}
+
+header {
+  display: flex;
```

Comment:
Don't use tag names for styling (except html and body)
<img width="823" alt="image" src="https://github.com/user-attachments/assets/9b885e15-3de1-4cdd-ace6-1d2b7bd83965">